HTTPServerProject.Tests/HTTPServerResponseTests/ByPathTests.cs

```diff
@@ -0,0 +1,365 @@
+using HTTPServerWriteTests.Streams;
+using HTTPServerResponse.Path;
+using HTTPServerResponseTests.Parameters;
+
+namespace HTTPServerResponseTests.PathTests;
+
+public class UnitTestsForResponsesByPath
+{
+    public List<string> eStream = new List<string>();
+
+    [Fact]
+    public void GetRedirectResponse()
+    {
+        var expected = new List<string>()
+        {
+            "HTTP/1.1 301 Moved Permanently", 
+            "Location: http://127.0.0.1:5000/simple_get", 
+            ""
+        };
+        var writer = new TestWriteStreams(eStream);
+        var path = "redirect";
+        var type = "GET";
+        var requestBody = "";
+        var pathParams = new TestPathParameters();
+        var pathDict = pathParams.pathDict;
+
+        var responsePath = new ResponsePath(writer, pathDict);
+        responsePath.ExecuteRequest(path: path, type: type, requestBody: requestBody);
```

Comment:
Would probably name this just `Execute` because Request has multiple meanings here.

Reply:
Agreed.